MobileCenterCrashes/MobileCenterCrashesTests/MSWrapperExceptionTests.m

```diff
@@ -0,0 +1,77 @@
+#import <Foundation/Foundation.h>
+#import <XCTest/XCTest.h>
+#import <OCHamcrestIOS/OCHamcrestIOS.h>
+#import <OCMock/OCMock.h>
+#import "MSWrapperExceptionInternal.h"
+#import "MSException.h"
+
+@interface MSWrapperExceptionTests : XCTestCase
+
+@property(nonatomic) MSWrapperException *sut;
+
+@end
+
+@implementation MSWrapperExceptionTests
+
+#pragma mark - Housekeeping
+
+- (void)setUp {
+  [super setUp];
+  self.sut = [self wrapperException];
+}
+
+#pragma mark - Helper
+
+- (MSWrapperException *)wrapperException {
+  MSWrapperException *exception = [MSWrapperException new];
+  exception.processId = [NSNumber numberWithInteger:4];
+  exception.exceptionData = [[NSData alloc] initWithBase64EncodedString:@"data string" options:NSDataBase64DecodingIgnoreUnknownCharacters];
```

Comment:
Do you really want Base64 string? Or just need an NSData object?
If you don't need Base64 string, just do `[@"data string" dataUsingEncoding:NSUTF8StringEncoding]`